### PR TITLE
add c-c++ package, disaster: disassemble c/c++ code

### PR DESCRIPTION
**Pull request by jb55**
https://github.com/jart/disaster

### CODE REVIEW

File: contrib/!lang/c-c++/packages.el [hunk 1]
```diff
@@ -43,11 +44,18 @@
...
         "mgA" 'projectile-find-other-file-other-window)
       (evil-leader/set-key-for-mode 'c++-mode
         "mga" 'projectile-find-other-file
+        "md" 'disaster
         "mgA" 'projectile-find-other-file-other-window))))
 
+(defun c-c++/init-disaster ()
+  (use-package disaster
+    :defer t
+    :commands (disaster)))
```
Comment (person808) [inline contrib/!lang/c-c++/packages.el]: It would be better for the keybindings to be set here to keep the package config seperated properly

Comment (jb55) [inline contrib/!lang/c-c++/packages.el]: :+1: updating

Comment (jb55) [inline contrib/!lang/c-c++/packages.el]: updated.

**Comment by person808** [comment]
:+1:

**Comment by robbyoconnor** [comment]
:+1: -- Love this.

**Comment by tuhdo** [comment]
You may be also interested in `asm-mode` that can also highlight C/C++ mixed with assembly nicely. DEMO.

**Comment by robbyoconnor** [comment]
:points_up:  :+1:

**Comment by syl20bnr** [comment]
Thank you :+1: 
Since `SPC m d` is defined in the conventions I moved the bindings for disaster to `SPC m D`
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**